### PR TITLE
feat(exoflex): support a11y for drawer

### PR DESCRIPTION
**Pull request by oshimayoan**
Add a11y support for native and web Drawer component.

### CODE REVIEW

File: packages/exoflex/src/components/Drawer/DrawerSection.tsx [hunk 1]
```diff
@@ -24,12 +37,24 @@ export default function DrawerSection(props: Props) {
     footerLabel,
     footerIcon,
     footerOnPress,
+    accessibilityLabel,
```
Comment (danielsukmana) [inline packages/exoflex/src/components/Drawer/DrawerSection.tsx:40]: This seems unused?

File: packages/exoflex/src/components/Drawer/DrawerSection.tsx [hunk 2]
```diff
@@ -24,12 +37,24 @@ export default function DrawerSection(props: Props) {
...
+    accessibilityRole,
+    footerAccessibilityLabel,
+    footerAccessibilityHint,
```
Comment (danielsukmana) [inline packages/exoflex/src/components/Drawer/DrawerSection.tsx:43]: These also seem unused?

File: packages/exoflex/src/components/Drawer/DrawerSection.tsx [hunk 3]
```diff
@@ -24,12 +37,22 @@ export default function DrawerSection(props: Props) {
     footerLabel,
     footerIcon,
     footerOnPress,
+    accessibilityRole,
+    footerAccessibilityLabel,
```
Comment (danielsukmana) [inline packages/exoflex/src/components/Drawer/DrawerSection.tsx:41]: Nit: missed this unused prop.

File: packages/exoflex/src/components/Drawer/DrawerSection.tsx [hunk 4]
```diff
@@ -38,18 +61,30 @@ export default function DrawerSection(props: Props) {
...
+      style={styles.container}
+    >
       <Header />
       <View style={{ flex: 1 }}>{children}</View>
       {footerLabel && (
         <DrawerItem
+          accessibilityRole={
+            footerAccessibilityRole || defaultFooterAccessibilityRole
+          }
+          accessibilityState={footerAccessibilityState}
```
Comment (danielsukmana) [inline packages/exoflex/src/components/Drawer/DrawerSection.tsx:87]: Anyway, I don't think we should remove `footerAccessibilityLabel` and `footerAccessibilityHint`. Instead, we should properly pass them on to this component along with other `footerAccesibility`-ies.

Comment (oshimayoan) [reply packages/exoflex/src/components/Drawer/DrawerSection.tsx:87]: Oops, missed that one.

**Pull request merged**